entries/c/calendly.com.json

```
@@ -1,11 +1,10 @@
 {
   "Calendly": {
     "domain": "calendly.com",
-    "contact": {
-      "twitter": "Calendly",
-      "facebook": "Calendly",
-      "email": "support@calendly.com"
-    },
+    "tfa": [
+      "totp"
+    ],
+    "documentation": "https://help.calendly.com/hc/en-us/articles/26682031653399-Two-factor-authentication-2FA",
```

Thanks for the pull request @mkimelblat!
Just need you to clean up the documentation URL and then your pr will be good to go! :smile: 
```suggestion
    "documentation": "https://help.calendly.com/hc/articles/26682031653399",
```